flocker/node/agents/blockdevice.py

```diff
@@ -230,6 +235,46 @@ def _action(self):
         return DESTROY_BLOCK_DEVICE_DATASET(_logger, volume=self.volume)
 
 
+# Implement:
+#     AttachVolume(volume=volume),
+#     MountVolume(volume=volume),
+# in order to carry out the sequence below.
+# Maybe use those in the CreateBlockDeviceDataset state change or paste in some
+# followup issues, or maybe do those issues first??
+
+
+@_logged_statechange
+@with_cmp(["change"])
+class ResizeBlockDeviceDataset(proxyForInterface(IStateChange, "change")):
```

Comment:
This idiom seems super-verbose and hacky compared to just having a utility function that creates the Sequentially things. See discussion in Zulip.